### PR TITLE
Show Empty State in Badges List for Mobile Views

### LINKED ISSUE #4610: Empty State for Badges Panel
**Is your feature request related to a problem? Please describe.**

The empty state for the badges panel (on mobile) when a user has no badges / repos linked is plain white.

94046D77-A633-4E08-8A98-6C4739446A62

**Describe the solution you'd like**

Add a little something to the section like “This is where all the badges you’ve collected go!” to motivate people to write and contribute more! (You guys can definitely think of a better copy for this!)

Let me know if I missed something. If ever, I’d love to take this on as well!

### PR DESCRIPTION
**Pull request by cesc1989**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description

Instead of an empty sidebar, add a text that is displayed in mobile views except desktop.

Left an ipsum as suggested by @jessleenyc in https://github.com/thepracticaldev/dev.to/issues/4610#issuecomment-549384992

## Related Tickets & Documents

Would close #4610 

## Mobile & Desktop Screenshots/Recordings

image

## Added to documentation?

- [x] no documentation needed

### CODE REVIEW

**Comment by benhalpern** [comment]
Thanks for this!

I'm not entirely sure why the styles for the existing code is inline like this, so I think it's cool that you followed the pattern.

I think this whole section could be refactored but I'm cool with following the pattern for now.

File: app/views/articles/_badges_area.html.erb [hunk 1]
```diff
@@ -54,4 +54,24 @@
...
+    @media screen and (min-width: 1024px) {
+      .no-badges-text {
+        display: none;
+      }
+    }
+  </style>
+  <div style="overflow: visible;padding-top:20px;">
+    <div class="widget-body">
+      <p class="no-badges-text">
+        Eat and than sleep on your face annoy the old grumpy cat, start a fight and then retreat to wash when i lose and i shall purr myself to sleep or run around the house at 4 in the morning
```
Comment (rhymes) [inline app/views/articles/_badges_area.html.erb]: to be nitpicky: `i` should be capitalized to `I`, correct?

Thanks for this @cesc1989!

**Comment by cesc1989** [comment]
> I'm not entirely sure why the styles for the existing code is inline like this

@benhalpern I was surprised as well. And yep, following the pattern felt like the "right" way.

File: app/views/articles/_badges_area.html.erb (hunk 1 above)
Comment (cesc1989) [reply app/views/articles/_badges_area.html.erb]: Agreed, @rhymes . Doing the change:)!

**Comment by mstruve** [comment]
@ludwiczakpawel what do you think about this? Something we want to push out or do you have a different idea for a blank badge page?

**Comment by ludwiczakpawel** [comment]
@mstruve thanks for ping. i think that makes sense for now. we will eventually address address this but for now we don't have established patterns for blank states..

**Pull request merged**